glambie/processing/processing_helpers.py

```diff
@@ -220,7 +223,8 @@ def prepare_seasonal_calibration_dataset(region_config: RegionRunConfig,
 
 def extend_annual_timeseries_if_outside_trends_period(annual_timeseries: Timeseries,
                                                       data_catalogue_trends: DataCatalogue,
-                                                      timeseries_for_extension: Timeseries) -> Timeseries:
+                                                      timeseries_for_extension: Timeseries,
+                                                      remove_trend: bool = True) -> Timeseries:
```

Comment:
Missing doc string for new parameter

Reply:
Also, can't see this used anywhere. It better to keep function signatures simpler when possible.

Reply:
yes, actually this is just a "might use that in the far future". Will remove the param!